### PR TITLE
The new CCPP suite FV3_GFS_v16b_ugwpv1 with GFSv16b (low resolution, C96_L127).

### PR DESCRIPTION
**Pull request by ValeryYudin-NOAA**
The new CCPP suite FV3_GFS_v16b_ugwpv1 and regression tests for warm_start and cold_starts
with GFSv16b (low resolution, C96_L127). The new suite represents the upgrade of uGWP (V1),CU-CIRES. It is capable to reproduce the realistic FV3GFS climate, including equatorial dynamics (QBO and SAO, 20-60 km)) and annual cycles in the extra-tropics and polar regions in the upper atmsosphere (20-80km). It can be executed and tested with the gsldrag sub-grid scale orography physics of gsl/develop for UFS-R2O planned activity in 2021.  

### Issue(s) addressed

issues to be closed with this PR in repositories : ufs-weather-model #347  issue
                                                                               fv3atm #222 issue
                                                                               ccpp-physics #542 issue

## Testing

How were these changes tested?     answer: hera
What compilers / HPCs was it tested with? answer:  intel
Are the changes covered by regression tests? answer: yes (new baseline for UFS-127L-C96 see ufsw-weather-model/tests) 
Have the ufs-weather-model regression test been run?  answer: hera-intel
- Will the code updates change regression test baseline? If yes, why?
- Yes because the UFS does bot have realistic tests for UFS-127L-C96 configuration
-  Please show the baseline directory below.
- rt.gsd_ugwpv1_1c.log:+ NEW_BASELINE=/scratch1/NCEPDEV/stmp4/Valery.Yudin/FV3_RT/REGRESSION_TEST_GSL_DEVELOP
rt.gsd_ugwpv1_1c.log:+ NEW_BASELINE=/scratch1/NCEPDEV/stmp4/Valery.Yudin/FV3_RT/REGRESSION_TEST_GSL_DEVELOP_INTEL
- Please commit the regression test log files in your ufs-weather-model branch
 create mode 100644 tests/rt.gsd_ugwpv1.1m.log
 create mode 100644 tests/rt.gsd_ugwpv1_1c.log

## Dependencies

If testing this branch requires non-default branches in other repositories, list them.
https://github.com/ValeryYudin-NOAA/ccpp-physics/tree/master
https://github.com/ValeryYudin-NOAA/ufs-weather-model/develop

Those branches should have matching names (ideally): first PR

Do PRs in upstream repositories need to be merged first?
first https://github.com/ValeryYudin-NOAA/ccpp-physics/tree/master
- waiting on noaa-emc/nems/pull/<pr_number>
- waiting on noaa-emc/fv3atm/pull/<pr_number>

### CODE REVIEW

File: gfsphysics/GFS_layer/GFS_typedefs.F90 [hunk 1]
```diff
@@ -7702,21 +7831,21 @@ subroutine interstitial_print(Interstitial, Model, mpirank, omprank, blkno)
...
-    write (0,*) 'sum(Interstitial%tau_ogw         ) = ', sum(Interstitial%tau_ogw         )
-    write (0,*) 'sum(Interstitial%tau_tofd        ) = ', sum(Interstitial%tau_tofd        )
-    write (0,*) 'sum(Interstitial%tau_ngw         ) = ', sum(Interstitial%tau_ngw         )
-    write (0,*) 'sum(Interstitial%zmtb            ) = ', sum(Interstitial%zmtb            )
-    write (0,*) 'sum(Interstitial%zlwb            ) = ', sum(Interstitial%zlwb            )
-    write (0,*) 'sum(Interstitial%zogw            ) = ', sum(Interstitial%zogw            )
-    write (0,*) 'sum(Interstitial%dudt_mtb        ) = ', sum(Interstitial%dudt_mtb        )
-    write (0,*) 'sum(Interstitial%dudt_ogw        ) = ', sum(Interstitial%dudt_ogw        )
-    write (0,*) 'sum(Interstitial%dudt_tms        ) = ', sum(Interstitial%dudt_tms        )
+! CIRES UGWP v0  VAY-2020 does not make any sense to print SUM
```
Comment (climbfuji) [inline gfsphysics/GFS_layer/GFS_typedefs.F90:7834]: I disagree. This is useful to detect b4b differences between runs etc.

**Comment by ValeryYudin-NOAA** [comment]
If you consider it to be important you can keep it, I don't know how to use
the total sum as diagnostics.

**Comment by ValeryYudin-NOAA** [comment]
Also

write (0,*) 'sum(Interstitial%gw_dudt         ) = ',
sum(Interstitial%gw_dudt         )
-    write (0,*) 'sum(Interstitial%gw_dvdt         ) = ',
sum(Interstitial%gw_dvdt         )
-    write (0,*) 'sum(Interstitial%gw_dtdt         ) = ',
sum(Interstitial%gw_dtdt         )
-    write (0,*) 'sum(Interstitial%gw_kdis         ) = ',
sum(Interstitial%gw_kdis

in my branch some variables, like above, are removed from
"typedefs.F90 and meta'

**Comment by climbfuji** [comment]
Merged as part of https://github.com/NOAA-EMC/fv3atm/pull/241.

**Pull request closed without merging**